conf/application.conf

```diff
@@ -100,6 +100,20 @@ microservice {
   }
 }
 
+controllers {
```

Comment:
These changes are fine, but could you change something else in this file:

When I run the app I get a warning: "play.filters.headers.contentSecurityPolicy is deprecated in 2.7.0.  Please use play.filters.csp.CSPFilter instead.".